src/lib/common/sem.cpp

```diff
@@ -424,13 +424,50 @@ int get_curl_context(const std::string& key, struct curl_context* pcc)
   return 0;
 }
 
+/* ****************************************************************************
+*
+* get_curl_context_new -
+*/
+static int get_curl_context_new(const std::string& key, struct curl_context* pcc)
+{
+  pcc->curl   = NULL;
+  pcc->pmutex = NULL;
 
+  pcc->curl = curl_easy_init();
+
+  if (pcc->curl == NULL)
+  {
+    LM_E(("Runtime Error (curl_easy_init)"));
```

Comment:
There is a function **curl_easy_strerror**, but it doesn't seem to support **curl_easy_init**, so

NTC